docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -0,0 +1,391 @@
+Elastic works with the Open Source communities leveraging OpenTelemetry to provide the best coverage.
+
+[discrete]
+[[ci-cd-jenkins]]
+=== Jenkins
+
+[discrete]
+[[ci-cd-install-jenkins]]
+==== Install the OpenTelemetry plugin
+
+. On the Jenkins UI, go to *Manage Jenkins* > *Manage Plugins*.
++
+[role="screenshot"]
+image::images/jenkins-plugin-manager.png[Jenkins Plugin Manager]
++
+. Click the *Available* tab, and search for *OpenTelemetry*.
+. Select the *OpenTelemetry* checkbox and then click *Download now and install after restart*.
++
+To verify that the plugin is installed, click the *Installed* tab, and search for *OpenTelemetry Plugin*.
+
+[discrete]
+[[ci-cd-configure-jenkins]]
+==== Configure the OpenTelemetry plugin
+
+The OpenTelemetry plugin needs to be configured to report data to an OpenTelemetry service.
+In addition, you will need the endpoint of the OpenTelemetry service, the type of authentication,
+and the access credentials.
+
+. On the Jenkins UI, go to *Manage Jenkins* > *Configure System*.
+. Go to the OpenTelemetry Plugin section.
+. Configure your OpenTelemetry endpoint and authentication using the Elastic APM Server URL and the APM Server authentication:
+
+* If using the Elastic APM secret token authorization, select a *Bearer Authentication Token*, and add the token
+as a Jenkins secret text credential.
++
+[role="screenshot"]
+image::images/configure-otel-plugin.png[Configure OTEL plugin]
++
+* If using the Elastic API Key authorization, define the *Header Authentications*:
+** Header name: `"Authorization"`
+** Header value: a secret text credential with the value of `"ApiKey an_api_key"` (`an_api_key` is the value of the secret key)
+
+. Go to *Add Visualisation Observability Backend* and define the URL for your {kib} server.
++
+[role="screenshot"]
+image::images/kibana-url.png[Define Kibana URL]
++
+. Finally, there are additional settings to configure:
+
+* Endpoint certificates to use in communications.
+* The service name and service namespace sent to the OpenTelemetry service.
+* Timeouts and batch process times.
+* Any steps you may want to omit from the data you send.
++
+WARNING: You can export the OpenTelemetry configuration as environment variables to use them with other tools like otel-cli,
+Ansible Otel plugin, and so on. If you enable this option, consider that you can potentially expose the credentials in
+the console output.
+
+To learn more about the integration of Jenkins with Elastic Observability, see https://plugins.jenkins.io/opentelemetry/[OpenTelemetry].
+
+[discrete]
+[[ci-cd-jenkins-dashbaords]]
+==== Install Jenkins dashboards in {kib}
+
+There are out of the box {kib} dashboards that help visualize some metrics for the CI/CD platform.
+
+Using the {kibana-ref}/dashboard-import-api.html[Import API] or the {kib} UI, you
+can https://github.com/jenkinsci/opentelemetry-plugin/tree/master/docs/dashboards/elastic[install dashboards]
+that are compatible with version 7.12 or higher.
+
```

Comment:
TBC, the section regarding the Kibana import and visualisation.

Reply:
@cyrille-leclerc , can you clarify what do you want to add here?

I took some screenshots

1) Import UI
<img width="1279" alt="Screenshot 2021-11-18 at 12 41 03" src="https://user-images.githubusercontent.com/2871786/142417862-1a2dda08-ec0e-42b1-816f-c2c0e615a8e1.png">

<img width="311" alt="Screenshot 2021-11-18 at 12 41 34" src="https://user-images.githubusercontent.com/2871786/142417897-2167cf14-ff43-4836-b05a-f037e7b0ec18.png">

2) Dashboard ready to be used

![image](https://user-images.githubusercontent.com/2871786/142418070-c6e1902b-e95c-4047-9991-63af9ab4cfa3.png)


3) Dashboard UI
<img width="2552" alt="Screenshot 2021-11-18 at 12 44 44" src="https://user-images.githubusercontent.com/2871786/142417934-68e9330e-b5bd-4d67-ba47-1b7672502006.png">

Reply:
See https://github.com/elastic/observability-docs/pull/1266/commits/9f99ebeebdf659e80ad9b34098cd9ebc3fb671c3

Reply:
I love the guidance through the GUI.
It's important to indicate users that they have to select "Check for Existing Object / Overwrite".
I'm wondering how much we should talk about the dashboard import API. Maybe it's enough to focus on the GUI based. approach or to have a short mention after the GUI based solution saying "It's also possible to import dashboards using the 'Import API'".